docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md

```diff
@@ -66,7 +66,7 @@ ZEEBE_CLIENT_CONFIG_PATH=/path/to/credentials/cache.txt
 
 ## How can I provide a custom Zeebe SSL certificate?
 
-You configured a custom SSL certificate in your (remote) Zeebe endpoint and want Web Modeler to accept that certificate.
+You configured a custom SSL certificate in your (remote) Zeebe cluster URL and want Web Modeler to accept that certificate.
```

Comment:
```suggestion
You configured a custom SSL certificate in your (remote) Zeebe deployment and want Web Modeler to accept that certificate.
```

Not sure what fits here. Maybe something like `deployment` | `instance`. Not sure how we usually refer to this.